pages/index/_chart.vue

```diff
@@ -21,7 +21,7 @@ export default {
     StockChart,
     MapChart,
     QuickMods,
-    SunburstChart
-  }
+    SunburstChart,
```

Comment:
Hmm. do the recommended lint conventions have trailing commas?

Reply:
it could be that my Vetur options might have this by default ?

Reply:
Ok, I put the eslint recommendation above (for the eslint file)